inference-engine/tests/functional/plugin/shared/src/behavior/stress_tests.cpp

```diff
@@ -0,0 +1,53 @@
+// Copyright (C) 2018-2020 Intel Corporation
+// SPDX-License-Identifier: Apache-2.0
+//
+
+#include "behavior/stress_tests.hpp"
+#include "ngraph_functions/subgraph_builders.hpp"
+
+namespace LayerTestsDefinitions {
+
+std::string MultipleAllocations::getTestCaseName(const testing::TestParamInfo<MultipleAllocationsParams>& obj) {
+    LayerTestsUtils::TargetDevice targetDevice;
+    unsigned int allocationsCount;
+
+    std::tie(targetDevice, allocationsCount) = obj.param;
+    std::ostringstream result;
+    result << "targetDevice=" << targetDevice << "_";
+    result << "allocationsCount=" << allocationsCount;
+    return result.str();
+}
+
+void MultipleAllocations::SetUp() {
+    std::tie(targetDevice, m_allocationsCount) = this->GetParam();
+    function = ngraph::builder::subgraph::makeSplitConvConcat();
+}
+
+TEST_P(MultipleAllocations, InferWorksCorrectAfterAllocations) {
+    SKIP_IF_CURRENT_TEST_IS_DISABLED()
+
+    ConfigurePlugin();
+
+    InferenceEngine::CNNNetwork cnnNet(function);
+    auto ie = PluginCache::get().ie();
+
+    std::cout << "Load the network " << m_allocationsCount << " times..." << std::flush;
+    for (int i = 0; i < m_allocationsCount; ++i) {
+        ie->LoadNetwork(cnnNet, targetDevice);
+    }
+
+    std::cout << "\nCheck inference.\n";
+
+    // Experiments demonstrated that 10 cycles are enough to reproduce the issue
+    int infersCount = 10;
+    for (int j = 0; j < infersCount; ++j) {
+        LoadNetwork();
+
+        std::cout << "Infer(): " << j << std::flush;
+
+        Infer();
+        Validate();
```

Comment:
Should it look like calling some base class method require number of times?

Reply:
I'm not sure. If this is only one place where it is necessary, then we should not create a new method. In the general case, from my point of view, it should be created as soon as this logic is duplicated. But....
Also, please pay attention, that I skipped part with this code:
```
SKIP_IF_CURRENT_TEST_IS_DISABLED()
ConfigurePlugin();
```
that contained in method `Run()`. So it should be like `LoadInferAndValidate(int n)`
Or it should be like a builder that builds a sequence of methods.